### ISSUE TITLE
Image cassée sur https://mon-entreprise.urssaf.fr/nouveaut%C3%A9s/septembre-2023

### ISSUE THREAD

**Issue by lorenzoalali** [betagouv/mon-entreprise#2928]
Sur https://mon-entreprise.urssaf.fr/nouveaut%C3%A9s/septembre-2023
la capture d'écran
l'image https://github.com/betagouv/mon-entreprise/assets/1775934/5abf6516-10e1-4360-9cc7-be47f33be1c7
n'apparait pas en dessous du lien "👉 Découvrir le nouvel assistant"
J'ai essayé plusieurs navigateurs différents. Le code semble correct
`<img alt="Screenshot 2023-09-04 at 17-08-00 Aide au choix du statut juridique" src="https://github.com/betagouv/mon-entreprise/assets/1775934/5abf6516-10e1-4360-9cc7-be47f33be1c7">`
J'imagine que c'est le fait que l'image soit hébergée sur GitHub qui pose problème, ou qu'il n'y ai pas d'extension .jpg ou .png à la fin du nom du ficher dans son URL d'origine.

![image](https://github.com/betagouv/mon-entreprise/assets/11087992/d836e6ac-c217-4699-ae73-a4fdb67ee142)

**Comment by lorenzoalali** [comment]
Problème similaire sur https://mon-entreprise.urssaf.fr/nouveaut%C3%A9s/f%C3%A9vrier-2023
![image](https://github.com/betagouv/mon-entreprise/assets/11087992/9723b729-95b2-437e-a281-3d697c4f9919)

**Comment by lorenzoalali** [comment]
Bonjour @johangirod Cette issue #2928 est marquée closed, mais le problème est toujours le même sur https://mon-entreprise.urssaf.fr/nouveaut%C3%A9s/septembre-2023

**Comment by johangirod** [comment]
Les images sont toujours cassées sur https://mon-entreprise.urssaf.fr/nouveaut%C3%A9s/juin-2024
C'est d'autant plus problématique que la release ne contient pas de texte pour décrire les nouvelles fonctionalités :p

Pour info j'ai un `NS_BINDING_ABORT` comme erreur lors de la requête.

![image](https://github.com/user-attachments/assets/78217075-ee71-493d-aac7-162b8c46a0d2)

@JalilArfaoui

**Comment by lorenzoalali** [comment]
Oui, même problème de mon côté. Aussi, une autre différence mineure est que la section "En bref" n'apparait pas dans une infobox en jaune contrairement aux mois précédents, et les titres de chaque section ne contiennent pas d'emoji comme précédemment.

![Screenshot_20240716_101236_Chrome](https://github.com/user-attachments/assets/89a1baa4-5d50-4434-a96b-f38dc565d45c)

![Screenshot_20240716_101207_Chrome](https://github.com/user-attachments/assets/30fd0cdc-4394-47d0-8c88-59ac23d205a2)

**Comment by JalilArfaoui** [comment]
Merci pour ces remontées, il faut en effet regarder ça !
